perception/ground_segmentation/docs/scan-ground-filter.md

```diff
@@ -28,16 +27,23 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
-| `split_height_distance`           | double | 0.2           | The height threshold to distinguishing far and near [m]                       |
-| `use_virtual_ground_point`        | bool   | true          | whether to use the ground center of front wheels as the virtual ground point. |
+| Name                              | Type   | Default Value | Description                                                                             |
+| --------------------------------- | ------ | ------------- | --------------------------------------------------------------------------------------- |
+| `input_frame`                     | string | "base_link"   | frame id of input pointcloud                                                            |
+| `output_frame`                    | string | "base_link"   | frame id of output pointcloud                                                           |
+| `global_slope_max_angle_deg`      | double | 8.0           | The global angle to classify as the ground or object [deg]                              |
+| `local_slope_max_angle_deg`       | double | 10.0          | The local angle to classify as the ground or object [deg]                               |
+| `radial_divider_angle_deg`        | double | 1.0           | The angle which divide the whole pointcloud to sliced group [deg]                       |
+| `split_points_distance_tolerance` | double | 0.2           | The xy-distance threshold to to distinguishing far and near [m]                         |
+| `split_height_distance`           | double | 0.2           | The height threshold to distinguishing far and near [m]                                 |
+| `use_virtual_ground_point`        | bool   | true          | whether to use the ground center of front wheels as the virtual ground point.           |
+| `detection_range_z_max`           | float  | 2.5           | Maximum height of detection range [m]                                                   |
+| `center_pcl_shift`                | float  | 0.0           | The distance from vehicle center of mass to the point using as center of pointcloud [m] |
+| `non_ground_height_threshold`     | float  | 0.2           | Height threshold of non ground objects [m]                                              |
+| `grid_mode_switch_radius`         | float  | 20.0          | The distance where grid division mode change from by distance to by vertical angle [m]  |
+| `grid_size_m`                     | float  | 0.5           | The first grid size [m]                                                                 |
+| `gnd_grid_buffer_size`            | uint16 | 4             | Number of grids using to estimate local ground slope                                    |
+| `elevation_grid_mode`             | bool   | true          | Elevation grid scan mode option                                                         |
 
 ## Assumptions / Known limits
 
```

Comment:
[imo] In "References/External links", could you add a reference to the paper you referred?

Reply:
I added here https://github.com/autowarefoundation/autoware.universe/commit/e6901d46b0ad0340bf474099a6eda3fac5f172b1